doc/release/1.18.0-notes.rst

```diff
@@ -14,6 +14,10 @@ New functions
 Deprecations
 ============
 
+Deprecate `np.alen`
+---------------------------------------------------------------
+`np.alen` was deprecated. Use `len` instead.
```

Comment:
You should put this in https://github.com/numpy/numpy/tree/master/changelog, with the filename `14181.deprecation.rst`. Our release note process changed very recently - sorry for the confusion